website/pages/docs/theming/component-style.mdx

```diff
@@ -159,32 +209,50 @@ passed, the `defaultProps` defined in the style config will be used.
 // 1. Import useStyleConfig
 import { useStyleConfig } from "@chakra-ui/react"
 
-function Button(props) {
-  const { size, variant, ...rest } = props
+function Card(props) {
+  const { variant, children, ...rest } = props;
+  const styles = useStyleConfig("Card", { variant });
 
-  // 2. Reference `Button` stored in `theme.components`
-  const styles = useStyleConfig("Button", { size, variant })
-
-  // 3. Pass the computed styles into the `sx` prop
-  return <Box as="button" sx={styles} {...rest} />
+  return (
+    <Box sx={styles} {...rest}>
+      {children}
+    </Box>
+  );
 }
 ```
 
-And lastly - the fun part - let's use our custom button component anywhere in
-our app:
+And lastly - the fun part - let's use our custom Card component anywhere in our app:
 
 ```jsx live=false
 // 1. Using the default props defined in style config
 function Usage() {
-  return <Button>Click me</Button>
+  return (
+    <Card>
+      <Image src="/lin.png" rounded="full" w={32} h={32} boxShadow="md" />
```

Comment:
When the user does the copy-pasta from the documentation, the images won't load, because they are not present on their system. 
A fully qualified image url could provide a better user experience.

Reply:
Sounds good. I'll add the placeholder image in the `public` folder and set the url to `https://chakra-ui.com/eric.jpg` for example.